easybuild/framework/easyblock.py

```diff
@@ -2894,6 +2901,15 @@ def prepare_step(self, start_dir=True, load_tc_deps_modules=True):
             '$ORIGIN/../lib64',
         ])
 
+        # Location to store RPATH wrappers
+        self.rpath_wrappers_dir = build_option('rpath_wrappers_dir')
```

Comment:
This build option should only be set on the command line, it should _not_ be part of the configuration or an environment variable. This is because if you did that the wrappers would never get re-created. I'm not sure how to enforce this.

Another option would be to empty the directory within the next `if` block by default.

Reply:
Hmm, I think I might be wrong here, it looks like the default behaviour is indeed to always overwrite any existing wrappers: https://github.com/easybuilders/easybuild-framework/pull/4596/files#diff-c1bad9950fee9f74b59e255119ea04ce41e9d9ce8cfa9fa94e831c8d314d106dR1074

Reply:
The only condition that would protect against this is https://github.com/easybuilders/easybuild-framework/pull/4596/files#diff-c1bad9950fee9f74b59e255119ea04ce41e9d9ce8cfa9fa94e831c8d314d106dR1055 and this is not triggered as the updates to `PATH` are not in place until later.

Reply:
I can add a check there to warn or error (or just log?) that the existing wrappers will be overwritten.